src/Dependencies/Roslyn/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "supports": {},
   "dependencies": {
-    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09"
+    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09",
+    "System.ValueTuple": "4.0.1-beta-24425-02"
```

Comment:
Well, I certainly hope it's a safe dependency, as Roslyn uses it. :). I've used throw expressions locally as well.